libraries/AP_NavEKF2/AP_NavEKF2_MagFusion.cpp

```diff
@@ -390,6 +391,17 @@ void NavEKF2_core::FuseMagnetometer()
         MagPred[1] = DCM[1][0]*magN + DCM[1][1]*magE  + DCM[1][2]*magD + magYbias;
         MagPred[2] = DCM[2][0]*magN + DCM[2][1]*magE  + DCM[2][2]*magD + magZbias;
 
+        if (pending_z_mag_correction && frontend->_mag_learn_limit > 0) {
+            pending_z_mag_correction = false;
+            float correction = magDataDelayed.mag[2] - MagPred[2];
+            const float correction_limit_ga = frontend->_mag_learn_limit*0.001*3;
```

Comment:
What is our style on constants??  To someone who is not familiar with the code, 0.001 and 3 have no real meaning to me.  Should these be #defined, const, or constexpr instead of just some hardcoded number??

Reply:
0.001 is for Gauss to milliGauss, 3 is the higher limit we are applying

Reply:
Thank you for the explanation.

But the issue is readability.  Magic numbers should have some kind of "constant" rather than just hardcoded.

Reply:
no, sorry. The code style we use in matternet must match that used in upstream ardupilot.
see my comment on GNC channel